tests/admin_views/tests.py

```diff
@@ -538,6 +538,18 @@ def test_sort_indicators_admin_order(self):
             self.assertContentBefore(response, 'The First Item', 'The Middle Item')
             self.assertContentBefore(response, 'The Middle Item', 'The Last Item')
 
+    def test_has_related_field_in_list_display(self):
+        """Ensure that joins are not performed for <FK>_id fields."""
```

Comment:
Just state the expected behavior: "Joins shouldn't be performed for ...." rather than "Ensure that" or "Test that" which is the purpose of all tests.